configuration/concepts/consultation_prenatale.csv

```diff
@@ -0,0 +1,51 @@
+6422a83f-acc6-4953-8239-f18a0b6d90a4,,CIEL:5596,Estimated date of delivery,Date prévue d'accouchement,Estimated date of delivery,Date prévue d'accouchement,,,Misc,Date,,,,,,,,,,,
+9e071d37-ee01-4785-a756-7e4059ffe736,,CIEL:1789,Gestational Age,Âge gestationnel,Gestational Age,L'âge gestationnel,,,ConvSet,N/A,,,Number of Weeks; Number of Days,,,,,,,,
+7ce5a9fe-d9e0-427e-a080-ce8a9c673822,,CIEL:1440,Fetal heart rate,Rythme cardiaque fœtal,Fetal heart rate,Rythme cardiaque fœtal,,,Misc,Numeric,,,,FALSE,beats/min,120,160,40,250,,
+f857c495-98d2-4097-9018-d6641a8304a5,,CIEL:1439,Fundal height,Hauteur utérine,Fundal height,Hauteur utérine,,,Misc,Numeric,,,,,cm,,,,,,
+c4526510-3f10-11e4-adec-0800271c1b75,,CIEL:160091,Cephalic,Céphalique,Cephalic,Céphalique,,,Misc,N/A,,,,,,,,,,,
+a0dc5727-c33e-479b-b70e-fb0eb7e45853,,CIEL:146922,Breech,Siège,Breech,Siège,,,Misc,N/A,,,,,,,,,,,
+5bb2ea3b-a736-4533-bd5e-a1e9d835f721,,CIEL:112259,Transverse,Transversale,Transverse,Transversale,,,Misc,N/A,,,,,,,,,,,
+cbafba29-b62a-4675-910a-a6bb161de256,,,Variable,Variable,Variable,Variable,,,Misc,N/A,,,,,,,,,,,
+d47e2e11-355e-4c62-86a7-8ebe45fb4292,,CIEL:160090,Presentation,Présentation,Presentation,Présentation,,,Misc,Coded,,Cephalic;Breech;Transverse;Variable,,,,,,,,,
+ec5216e0-aa54-41e5-8f8a-440295734cb4,,CIEL:115491,Multiple pregnancy,Grossesse multiple,Multiple pregnancy,Grossesse multiple,,,Misc,Coded,,Yes;No,,,,,,,,,
+2c1e2efe-74cb-475d-a922-3785e02e7bd3,,CIEL:147232,Vaginal bleeding,Saignements vaginaux,Vaginal bleeding,Saignements vaginaux,,,Misc,N/A,,,,,,,,,,,
+4ba98e11-cb3b-4a22-be4e-bb43f3078683,,CIEL:163750,Uterine contractions,Contractions utérines,Uterine contractions,Contractions utérines,,,Misc,N/A,,,,,,,,,,,
+7f703275-777c-4237-9d07-36c99ea0966c,,CIEL:153551,Abdominal pain,Douleurs abdominales,Abdominal pain,Douleurs abdominales,,,Misc,N/A,,,,,,,,,,,
+6f8988d1-157a-44cc-ac3b-22b627a57db3,,,Fever,Fièvre,Fever,Fièvre,,,Misc,N/A,,,,,,,,,,,
+e90fa130-09cd-4574-936f-91fca52a9324,,CIEL:125198,Swollen feet,Pieds gonflés,Swollen feet,Pieds gonflés,,,Misc,N/A,,,,,,,,,,,
+8d83ff2b-3f6f-432a-b0cd-5b4f2f1055fd,,CIEL:118939,Vision problems,Troubles de la vision,Vision problems,Troubles de la vision,,,Misc,N/A,,,,,,,,,,,
+f7312165-7e96-4761-b4cf-d601a1a18f9c,,CIEL:1727,Signs and Symptoms,Signes et symptômes,Signs and Symptoms,Signes et symptômes,,,Misc,Coded,,Vaginal bleeding;Uterine contractions;Abdominal pain;Fever;Swollen feet;Vision problems;Other,,,,,,,,,
+2f191568-b58f-4542-a69e-441209e88ea7,,,Other Signs and Symptoms,Autres signes et symptômes,Other Signs and Symptoms,Autres signes et symptômes,,,Misc,Text,,,,,,,,,,,
+da18a735-7f7d-4f77-a01c-d6b2ecfc36fe,,,Signs and Symptoms Data,Données sur les signes et symptômes,Signs and Symptoms Data,Données sur les signes et symptômes,,,Concept Details,N/A,,,Signs and Symptoms; Other Signs and Symptoms; Duration; Time Units,,,,,,,,
```

Comment:
"Signs and Symptoms Data" do we need the "data" here?
"Données sur les signes et symptômes" doesn't make much sense in french. Hopefully we can remove the 'data'.
Should we replace with "Signs and Symptoms Set"?

Reply:
Yes,sure